### PR TITLE
Update README.md and package.json

### PR DESCRIPTION
**Pull request by dannytranlx**
Renamed `npm run develop` to `npm run dev` to match our other projects and made it run both frameworks (Gastby and Slate)
Updated README.md by re-adding previous introduction and reformatting the setup commands

Also added a section about legacy Slate docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,30 @@
 # Smooch
+*Beautifully simple in-app messaging*
```
Comment (jpjoyal) [inline README.md]: Change to our current tag line "A complete platform to add multi-channel messaging to your software or business"

File: README.md [hunk 2]
```diff
@@ -1,13 +1,30 @@
...
 
-TODO : Improve this README.
+This project hosts the documentation available at [https://docs.smooch.io/](https://docs.smooch.io/).
 
-## Get started
+The docs provides in-depth information on Smooch’s features and is meant to be a guide for developers integrating the SDK and app makers seeking to engage with their users.
```
Comment (jpjoyal) [inline README.md]: app makers -> businesses

File: README.md [hunk 3]
```diff
@@ -1,13 +1,30 @@
...
 
-- Clone the repository
-- Install gatsby (`npm install -g gatsby`)
-- Install dependencies (`npm install` in the repo folder)
-- Run `gatsby develop` to run the project in dev mode.
+All source code examples are provided in Objective-C, Swift, and JavaScript! You can view code examples inline.
+
+For more in-depth details on Smooch’s api check out our [API reference](https://docs.smooch.io/rest/). The API reference documents every method and property available through Smooch. If you have any other questions, feel free to reach out to us [here](mailto:help@smooch.io).
+
+Smooch is an SDK that connects you to your users so you can communicate with them and build a killer app. If you want to learn more about Smooch, then visit [our website](https://smooch.io) or [sign up here](https://app.smooch.io/signup).
```
Comment (jpjoyal) [inline README.md]: I would remove this one. Its covered by the tagline suggested above

File: README.md [hunk 4]
```diff
@@ -30,10 +47,29 @@ section: SECTION_ID                 // see note below
...
+
+Run the server
+
+    $ bundle exec middleman server
+
+Those docs will be available on [http://localhost:4567](http://localhost:4567).
+
+**Incompability with Gastby docs**
+
+Unfortunately, they aren't properly linked to the current Gastby docs. Whenever you try to reach a link to the REST docs from the main docs, you will be redirected to the docs in [production](https://docs.smooch.io/rest), instead you need to type in [http://localhost:4567/](http://localhost:4567/) followed by the hash you want to navigate to.
```
Comment (jpjoyal) [inline README.md]: "Unfortunately, they aren't properly linked to the current Gastby docs" -> "Links to Legacy docs from Gastby don't work in the local environment"

File: README.md [hunk 5]
```diff
@@ -1,13 +1,28 @@
...
+
+For more in-depth details on Smooch’s API check out our [API reference](https://docs.smooch.io/rest/). The API reference documents every method and property available through Smooch. If you have any other questions, feel free to reach out to us [here](mailto:help@smooch.io).
+
+## Get Started
+
+Clone the repository and install the dependencies
+
+    $ git clone git@github.com:smooch/smooch-docs.git
+    $ cd smooch-docs
+    $ npm install -g gatsby
```
Comment (lemieux) [inline README.md]: I think installing gastby globally is pointless if we run `npm run dev`

Comment (dannytranlx) [inline README.md]: but `npm run dev` runs `gatsby develop` 👀

Comment (lemieux) [inline README.md]: exactly my point. when you run `npm run something` whatever that's run will pick in the local node modules first.

Comment (lemieux) [inline README.md]: it puts `node_modules/.bin` on the path

Comment (dannytranlx) [inline README.md]: ohhh it's like `bundle exec`

**Comment by dannytranlx** [comment]
@spasiu @gozman The new README recommends to use `npm run dev` to avoid installing `gatsby` or `middleman` as global dependencies, but if you have them already, you can keep using `gatsby develop` or `middleman server` without any issues. 

You can also use the new npm script if you also have `middleman` installed already 🙌

**Pull request merged**